Rakefile

```diff
@@ -20,6 +21,9 @@ Hoe.spec "flay" do
   dependency "ruby_parser",    "~> 3.0"
   dependency "erubis",         "~> 2.7.0"
 
+  dependency "minitest",       "~> 5.8.0", :dev
+  dependency "ruby2ruby",      "~> 2.2.0", :dev
```

Comment:
I agree that ruby2ruby is optional at runtime, but it should be required for development since tests rely on it being present.

Fair on minitest, I guess, but again it's required for development, while hoe-seattlerb is optional. Some of us (*cough*) don't have it installed. ;)

And introducing the hoe-bundler plugin is by definition an optional thing, only for people who *do* use bundler.

What would you like me to do here?